pennylane/devices/qutrit_mixed/measure.py

```diff
@@ -91,24 +98,35 @@ def calculate_reduced_density_matrix(
 
 
 def calculate_probability(
-    measurementprocess: StateMeasurement, state: TensorLike, is_state_batched: bool = False
+    measurementprocess: StateMeasurement,
+    state: TensorLike,
+    is_state_batched: bool = False,
+    measurement_errors=None,
```

Comment:
nitpick (non-blocking): type hinting isn't used here. not sure what the package convention is for things that default to None.